FirebaseAnalyticsSwift/Tests/SwiftUnit/AnalyticsAPITests.swift

```diff
@@ -56,11 +67,15 @@ final class AnalyticsAPITests {
 
     // MARK: - Consent
 
-    Analytics.setConsent([.analyticsStorage: .granted, .adStorage: .denied])
+    Analytics.setConsent([.adPersonalization: .granted,
+                          .adStorage: .denied,
+                          .adUserData: .granted,
+                          .analyticsStorage: .denied])
```

Comment:
Could you please add a switch that will assert that all expected cases exist. Here is an example of a pattern I used: 

https://github.com/firebase/firebase-ios-sdk/blob/770776d19f3270ba708081f6bd607136e2dbd132/FirebaseInAppMessaging/Swift/Tests/Unit/FirebaseInAppMessaging_APIBuildTests.swift#L67-L75

This way, in case the cases were to unintentionally change, this file would fail to build.

Reply:
Done, thanks a lot for the suggestion!